include/inviwo/core/processors/processorutils.h

```diff
@@ -111,6 +113,23 @@ std::unique_ptr<T> makeProcessor(ivec2 pos, Args&&... args) {
     return p;
 }
 
+/**
+ * @brief Find which module that registered a processor
+ * @param processor the processor to look for
+ * @param app the InviwoApplication needed to get the modules
+ * @return the InviwoModule that registered the processor or nullptr if not found
+ */
+IVW_CORE_API InviwoModule* getProcessorModule(Processor* processor, const InviwoApplication& app);
```

Comment:
should be const Processor*
Implementation should check that processor is not null.

Reply:
fixed

Reply:
No changes pushed yet...

Reply:
Still not changed

Reply:
that was very strange, I had fixed it locally and had now changes according to git, but it still was not committed, I restarted VS and the changes was gone. seems VS did something strange...
Should be fixed now